en/includes/guides/authentication/mfa/add-push-auth-login.md

```diff
@@ -55,29 +55,6 @@ To update the default Push Notification settings:
 
 3. Once you update the Push Notification settings, click **Update**.
 
-## Configuring Push Providers
```

Comment:
Now that this section has removed from push auth login docs, shall we add a note here, asking user to configure push provider before proceeding to add push auth to login sequence?

Reply:
https://github.com/wso2/docs-is/pull/5308/files#diff-6da1c31902fa0862047fc573eb1966bf34959c4a133a6275fb928922295c32eeR12 This is already added in prerequisites